pages/common/nano.md

```diff
@@ -3,22 +3,26 @@
 > Simple, easy to use command-line text editor. An enhanced, free Pico clone.
 > More information: <https://nano-editor.org>.
 
+- Open nano without a file:
+
+`nano`
+
 - Open a specific file:
 
 `nano {{path/to/file}}`
 
-- Open a file positioning the cursor at the specified line and column:
+- Open a specific file positioning the cursor at the specified line and column:
 
 `nano +{{line}},{{column}} {{path/to/file}}`
 
-- Enable smooth scrolling:
+- Open a specific file and enable smooth scrolling:
 
-`nano -S {{filename}}`
+`nano -S {{path/to/file}}`
 
-- Indent new lines to the previous lines' indentation:
+- Open a specific file and indent new lines to the previous lines' indentation:
 
-`nano -i {{filename}}`
+`nano -i {{path/to/file}}`
 
-- Before modification, backup separately as `{{current_file_name}}~`:
+- Open nano and create a backup if edits are saved:
```

Comment:
```suggestion
- Open nano and create a backup file (`file~`) if edits are saved:
```
What do you think?

Reply:
```suggestion
- Open nano and create a backup file {{file}}~ if edits are saved:
```

Reply:
Most clients will mess up token syntax inside of descriptions